web-client/src/applicationContextPublic.js

```diff
@@ -94,6 +110,24 @@ const applicationContextPublic = {
   getConstants: () => frozenConstants,
   getCurrentUserToken: () => null,
   getHttpClient: () => axios,
+  getLogger: () => ({
+    error: () => {
+      // eslint-disable-next-line no-console
+      // console.error(value);
```

Comment:
Should this be updated instead of commented out?

Reply:
The logger for the clients is currently just a placeholder until we implement https://github.com/ustaxcourt/ef-cms/issues/457